### PR TITLE
Icons for navbar

### LINKED ISSUE #3328: Change navbar links to icons
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

To make the Navbar more compact replace the following navbar links:
1. `profile` link with the icon of the person
2. `Dashboard` link with the house icon
3. We want to hear about this point from you. Suggest which other navbar links can be changed :thinking: 

Hint: You need to make changes in this https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb file 

Mention @publiclab/community-reps after solving as getting their consent is essential for choosing icon and for finalizing the view too. Feel free to ask for help, if required. Thanks!

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Fixes #3328 
@publiclab/reviewers Please review and approve..
Added icons for `Dashboard` and `Profile`..

screenshot 67

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -120,6 +120,7 @@
                 <li><a href="/logoutRemotely">Logout from all devices</a></li>
               </ul>
             <% else %> <!-- Login button -->
+		return_to=<%= params[:return_to] || request.path %>
```
Comment (SidharthBansal) [inline app/views/layouts/_header.html.erb]: Please remove it

**Comment by oorjitchowdhary** [comment]
@SidharthBansal done

**Comment by oorjitchowdhary** [comment]
@HarshKhandeparkar I don't think it'd look good if I put the text down.. Sorry, but I think icons on the right look good.. let mentors review this ..
@publiclab/reviewers

**Comment by SidharthBansal** [comment]
Please claim the task and get some rewards

**Comment by SidharthBansal** [comment]
@jywarren merging this one. Hope you will like it.

**Comment by oorjitchowdhary** [comment]
> Please claim the task and get some rewards

Please create a task for this., it isn't there on gci.. Thanks

**Comment by SidharthBansal** [comment]
I published the task 
https://github.com/publiclab/plots2/issues/3328#issuecomment-431872677
Please check it again.

**Pull request merged**